### ISSUE TITLE
Just question.

### ISSUE THREAD

**Issue by username1565** [naxuroqa/Venom#496]
Do you have venom.zip + source code, from 2014 year, which was been specified here: https://habr.com/ru/post/212029/ ?

Thanks.

**Comment by username1565** [comment]
This is not issue, just question.

**Comment by naxuroqa** [comment]
Hey @username1565 there is still a branch which points to the version before the UI refactoring.

https://github.com/naxuroqa/Venom/tree/legacy

It's probably not that easy to get it to build.
I gave up on trying to make the app to look like the mock-up as GTK+-3.0 does not officially support any theming (apart from small changes). It's very unstable and even minor GTK releases will break your theme.

**Comment by username1565** [comment]
No, no. I don't mean the user interfase. I mean the first version of win32 `.exe` file, which was been specified on `habr.com`, by that link.
As I understand, this is the first version of `desctop`-application `Tox` for Windows, and after this was been developed the `uTox`, right?
So, I just wanted to get this as binary, and compile this from the source code.
Because `uTox` have the many dependencies, which are too old, and too many updated now.
So, old versions of `uTox` cann't be compiled.

Thelefore, I did ask you, about... Do you have portable venom.exe, as working p2p program for windows. And the working source code will be a great bonus for this.

Best regards.
